### PR TITLE
fix width_shift_range and height_shift_range in ImageDataGenerator

### PR DESCRIPTION
**Pull request by old-school-kid**
[Error notebook](https://colab.research.google.com/drive/1mUttIvC3f-RAbSQcAfPvQ8bI5u3K4Yoi?usp=sharing)

[Line 1915](https://github.com/keras-team/keras/blob/master/keras/preprocessing/image.py#L1915) checks for height shift but changes tx and likewise for width shift and ty. 

cc @qlzh727

### CODE REVIEW

**Review by fchollet — COMMENTED** [review]
Thanks for the PR. Please add a unit test. It should be a test that would have failed before this PR but that passes after this PR.

**Comment by old-school-kid** [comment]
@fchollet can you please let me know a way to add a unit test for this since it shifts image in a random fashion and there is no provision to have a seed for this. The error can be visualized in the colab link I have attached though. 
TIA

**Comment by fchollet** [comment]
> there is no provision to have a seed for this

Can you just do `keras.utils.set_random_seed(123)`?

**Comment by old-school-kid** [comment]
@fchollet could you please take a look at this?

This [notebook](https://colab.research.google.com/drive/1mUttIvC3f-RAbSQcAfPvQ8bI5u3K4Yoi?usp=sharing) contains the detailed error and the test case that I added which fails with original implementation but passes after the patch. 

Only added test for `width_shift` and not for `vertical_shift` since the variables were interchanged only.

**Comment by old-school-kid** [comment]
@fchollet could you take a look at this?
Thank you in advance!

**Review by fchollet — APPROVED** [review]

**Comment by old-school-kid** [comment]
@fchollet Would you please let me know what tests are failing?
